.dockerignore

```diff
@@ -0,0 +1,13 @@
+.git
+.husky 
+.github
+.gitignore
+.gitpod.yml
+.prettierrc 
+.eslintrc.json 
+
+LICENSE
+README.md
+Dockerfile
+CONTRIBUTING.md
+CODE_OF_CONDUCT.md
```

Comment:
Missing an endline.
```suggestion
CODE_OF_CONDUCT.md

```

Reply:
We generally don't add a blank line in the `.dockerignore`, no worries added it.